src/PageCharacter/CharacterDisplay/Tabs/TabOverview/index.tsx

```diff
@@ -0,0 +1,154 @@
+
+export default function TabOverview() {
+  const scrollRef = useRef<HTMLDivElement>()
+  const onScroll = useCallback(() => scrollRef?.current?.scrollIntoView?.({ behavior: "smooth" }), [scrollRef])
+
+  return <Stack spacing={1}>
+    <Grid container spacing={1} sx={{ justifyContent: "center" }}>
+      <Grid item xs={8} sm={5} md={4} lg={2.3}  >
+        <CharacterProfileCard />
+      </Grid>
+      <Grid item xs={12} sm={7} md={8} lg={9.7} sx={{ display: "flex", flexDirection: "column", gap: 1 }} >
+        <EquipmentRow onClick={onScroll} />
+        <CardLight sx={{ flexGrow: 1, p: 1 }}>
+          <StatDisplayComponent />
+        </CardLight>
+      </Grid>
+    </Grid >
+    <Box ref={scrollRef}>
+      <EquipmentSection />
+    </Box>
+  </Stack>
+}
+function EquipmentRow({ onClick }: { onClick: () => void }) {
+  const { character: { equippedWeapon } } = useContext(CharacterContext)
+  const { data, } = useContext(DataContext)
+
+  return <Grid container spacing={1} columns={{ xs: 2, sm: 2, md: 3, lg: 6, xl: 6 }}>
+    <Grid item xs={1}>
+      <WeaponCardNano weaponId={equippedWeapon} BGComponent={CardLight} onClick={onClick} />
+    </Grid>
+    {allSlotKeys.map(slotKey =>
+      <Grid item key={slotKey} xs={1} >
+        <ArtifactCardNano artifactId={data.get(input.art[slotKey].id).value} slotKey={slotKey} BGComponent={CardLight} onClick={onClick} />
+      </Grid>)}
+  </Grid>
+}
+/* Image card with star and name and level */
+function CharacterProfileCard() {
+  const { characterSheet, character: { key: characterKey, team } } = useContext(CharacterContext)
+  const { data, } = useContext(DataContext)
+  const characterDispatch = useCharacterReducer(characterKey)
+  const navigate = useNavigate()
+  const charEle = data.get(input.charEle).value as ElementKey
+  const weaponTypeKey = characterSheet.weaponTypeKey
+  const level = data.get(input.lvl).value
+  const ascension = data.get(input.asc).value
+  const constellation = data.get(input.constellation).value
+  const tlvl = {
+    auto: data.get(input.total.auto).value,
+    skill: data.get(input.total.skill).value,
+    burst: data.get(input.total.burst).value,
+  }
+  const tBoost = {
+    auto: data.get(input.bonus.auto).value,
+    skill: data.get(input.bonus.skill).value,
+    burst: data.get(input.bonus.burst).value,
+  }
+  const [{ favorite }, setCharMeta] = useDBState(`charMeta_${characterKey}`, initCharMeta)
+  return <CardLight sx={{ height: "100%" }} >
+    <Box sx={{ position: "relative" }}>
+      <Box sx={{ position: "absolute", width: "100%", height: "100%" }}>
+        <Typography variant="h6" sx={{ position: "absolute", left: "50%", bottom: 0, transform: "translate(-50%, -50%)", opacity: 0.75 }}>
+          <StarsDisplay stars={characterSheet.rarity} colored />
+        </Typography>
+        <Chip color={charEle} sx={{ height: "auto", position: "absolute", left: "50%", bottom: "5%", transform: "translate(-50%, -50%)", opacity: 0.85, maxWidth: "100%" }}
```

Comment:
Some funny looking wrapping with Hu Tao 
<img width="223" alt="image" src="https://user-images.githubusercontent.com/36019388/181636460-96fa4524-6205-4205-b5b5-17dd6ab03412.png">

Also, should the order be <element> <weapon> <name> instead?

Reply:
`element weapon name` looks weird. 
![image](https://user-images.githubusercontent.com/1754901/181660562-75c0ea0a-d404-4888-bca9-ae92a4baa3eb.png)
Text does not look "centered"